NukeOps gun renovation with bundles

LanCartwright opened the pull request:
Changed gun prices, mech and borg prices.
Adds more bundles.
Slashes several prices.

Will comment out Bulldog related content if needed.

:cl:
tweak: The TC costs for nuke ops have been rebalanced. Guns, viscerators and mechs are cheaper, ammo and borgs are more expensive.
add: Most guns can now be bought in bundles, together with some ammo and freebies for a special discount.
/:cl:

LanCartwright committed:
For a 60 crew round, this theoretically allows all operatives to own a gun and some ammo, and still have some points left over for luxuries.

This mainly balances gun prices and adds new bundles

LanCartwright committed:
dsfdf

LanCartwright committed:
Update backpack.dm

LanCartwright committed:
Update projectile.dm

Chronitonity commented on code/datums/uplink_item.dm:
```diff
@@ -127,11 +161,19 @@ var/list/uplink_items = list()
 	surplus = 50
 
+/datum/uplink_item/dangerous/shotgun
+	name = "C-20r Submachine Gun"
```
Probably meant to be bulldog

LanCartwright replied:
oops. nice catch. thanks

LanCartwright committed:
Update uplink_item.dm

Lumipharon commented:
For the love of god don't make mechs cheaper.
Having one dude who is effectively immune to everything except the ion rifle slowly grind the entire crew out of existence is already strong enough as is.

Also while the gun changes aren't a bad idea, the root problem of having a very effective and free gun by default isn't really addressed.

LanCartwright commented:
What I meant by "comment out bulldog related" was a statement that if people *really* want the bulldog to be default despite discussions, I will comment it out so they can do so.

The idea of having a free gun is moot after this change, and it is addressed by slashing the prices of guns and promoting people to purchase one.

As for mechs, it is due to a discussion that few people buy mechs, and even worse when the Borg is in every way valued more than the Mauler

LanCartwright committed:
whoops that was a mistake

Lumipharon commented:
the borg change is fine - borgs are an additional player with all access.
A mech, while retardly robust and destructive, requires an op to pilot it.

That said, if ops buy a mauler and raid the armoury first, 9 times out of 10 the mech can just kill the entire crew single handedly.
Mechs aren't bought often because it's a rambo item that means the rest of the team gets shit all, not because it's too expensive to be good.

LanCartwright commented:
hm, yeah. I see your point now. I just redid the math. You're right about that.

Do you want me to reduce the Gygax price too, or are you concerned mostly about the Mauler price?

Lumipharon commented:
The mauler is far, far better then the gygax. It has burst fire 20 brute shots instead of the gax's incendiary weaksauce. It's also rediculously tanky compared to the gax.

The mauler shouldn't be cheaper, but with your new gygax price, it's more then possible to see double mech on busier rounds, which would be a tad amusing.

LanCartwright committed:
reverts mauler to previous price.

TheNightingale commented:
I like it. I'll like it even more if you add a sniper bundle.
It's M-90gl, though, not C-90gl.
The biggest discount is the Medical bundle (save 5TC, or +33%) - remember the traitor's surplus Syndiecrate gives you 40-50 TC of loot for 20 TC (save 25TC, or +125%). Perhaps pad the bundles out a little with more stuff (emags and C4 are good - maybe a flamer for the Bulldog bundle)?

Lumipharon commented:
surplace crates have random shit, which is why they give high amounts of stuff.
Actual traitor bundles usually only have around 24tc worth of shit.

tkdrg commented on code/datums/uplink_item.dm:
```diff
@@ -215,7 +257,7 @@ var/list/uplink_items = list()
 	desc = "A unique grenade that deploys a swarm of viscerators upon activation, which will chase down and shred any non-operatives in the area."
 	item = /obj/item/weapon/grenade/spawnergrenade/manhacks
-	cost = 8
+	cost = 5
```
Reeeally not sure about this one. Viscerators are already incredible value for the price, but I'm willing to give it a try at 5TC and see how it goes.

tkdrg commented:
A bit disappointed at the lack of a sniper bundle, but this is a cool idea :+1:

Niknakflak commented:
:-1: by adding magazines and such, you aren't slashing the price of just the gun but you gotta take into account the entire discount of everything in the bundle

If that's already the case
:+1:
If not
Fix it

LanCartwright commented:
Yeah I already compensated the C20r price.

This renovation makes it extremely reasonable to purchase a gun, but I compensated for the cheaper pricing by raising the magazine price a little more.

What this essentially means is, buying a gun doesn't nuke your TCs.
However, if you want to have a fuckton of magazines, the price start's catching up.

Everything in the bundle is discounted. It is more expensive to purchase them individually in every case.

An emphasis on buying these bundles is to create oppotunities to try out the less used items.

I'll get a sniper bundle up, with a worn out suit and tie.

Steelpoint commented:
Wouldn't this PR do better with removing the round start Bulldogs or whatever?

Steelpoint commented:
Without removing the free round start Bulldog then this PR won't really change much, why buy a gun crate when you already start with a powerful weapon from the get go?

LanCartwright commented:
It is implied that I'll have Bulldogs removed, but compensate by making them extremely cheap to purchase.

Steelpoint commented:
I suggest putting them to the same price as a C20r, you can get a SMG with low bullet damage but high stamina damage, or a powerful auto shotgun with a wide range of versatile ammo types.

Or pony up more for a 3 hit kill rifle.

LanCartwright commented:
Someone can purchase a bulldog and get one drum magazine free for the price of a C20r. I think that's currently fairly balanced considering one weapon has 65 stamina damage per bullet, not to mention a two shot burst.

In comparison, c20r for 10 TC
20 rounds equal to 10 bursts, 1300 stamina damage and 400 damage.

And the bulldog, for 8 TC.
8 rounds, 60 damage a round, and 480 damage total.

Given the price of a bulldog and another slug drum, the damage total would be, for 10 TC;
16 rounds, 60 a round, 960 damage total.

I think the prices are fair, for now. It wouldn't surprise me if someone wants the Bulldogs to use slugs instead of buckshot now that they are no longer roundstart, hence reducing the "friendly fire problem".

Supermichael777 commented:
buldogs ship with slugs roundstart because frendly fire sucks

LanCartwright committed:
adds so much more shit

LanCartwright committed:
Merge branch 'nukeopsrenovation' of https://github.com/LanCartwright/-tg-station into nukeopsrenovation

# Conflicts:
#	code/game/objects/items/weapons/storage/backpack.dm

LanCartwright committed:
merge error

LanCartwright committed:
merge error

LanCartwright committed:
solves travis

Lumipharon commented:
daily reminder to add meteor shot (or a nerfed version of it) to the bulldog to let them blow past doors that the AI locks down.

LanCartwright committed:
Changes c90gl to m90gl

LanCartwright committed:
missed this one

LanCartwright commented:
Changelog

Added to uplink:

Bundles:
C20r bundl - 14 TC.
M90gl bundle - 15 TC.
Bulldog bundle - 16 TC.
Medical bundle - 15 TC.
Sniper bundle - 20 TC.
Chemical bundle - 30 TC.
C20r ammo bundle - 20 TC
Fireteam (c20r, m90gl, 40mm grenade, sniper rifle) ammo bundle - 24 TC.

Items:
Bulldog shotgun - 8 TC.

Added items:
Cheap suppressor.
Tactical turtleneck suit.

Price changes (Reductions only):
C20r - 14 to 10 TC.
M90gl - 18 to 12 TC.
Toy C20r - 8 to 5 TC.
Toy L6 SAW - 12 to 10 TC.
Viscerator delivery grenade - 8 to 5 TC.
Dark Gygax exosuit - 90 to 80 TC.

Price changes (Increases only)
Syndicate Cyborg - 50 to 80 TC
C20r magazine - 2 to 3 TC.
Bulldog ammo bag - 10 to 12 TC.
M90gl magazine - 2 to 4 TC.
40mm grenades - 4 to 5 TC.


Items in bundles are described in their description.

tkdrg commented:
Alright then, I think this looks good, so let's give it a try.
